pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[tool.poetry]
+name = "perceval"
+version = "0.17.1"
+description = "Fetch data from software repositories"
```

Comment:
I'd like to change this to `Send Sir Perceval on a quest to fetch and gather data from software repositories.`.